cypress/fixtures/item_barcodes9.csv

```diff
@@ -0,0 +1,9 @@
+"4539876054383"
+"453987605438"
+"4539876054382"
+"BW-item1"
+"BW-item2"
+"12"
+"X575181"
+"10101"
+"90000"
```

Comment:
you need to create a set of items for every test, see `cypress\e2e\bulk-edit\in-app\bulk-edit-in-app-item-barcodes.cy.js`